e2e/src/lib.rs

```diff
@@ -92,6 +92,17 @@ mod tests {
         let show_logs = false;
         let blob_support = true;
         let stack = WholeStack::deploy_default(show_logs, blob_support).await?;
+        let db_clone = stack.db.clone();
```

Comment:
This is copied from [here](https://github.com/FuelLabs/fuel-block-committer/pull/188) because the e2e tests were flaky. Will solve the conflict once the above PR gets merged.